test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -664,9 +664,12 @@ def test_maxpool2d_single_padding(self):
         model = nn.MaxPool2d(5, padding=2)
         self.run_model_test(model, train=False, batch_size=BATCH_SIZE)
 
-    @unittest.skip("C2 and PyTorch have small difference in padding implementation")
     def test_avgpool2d(self):
-        model = nn.AvgPool2d(5, padding=(2))
+        model = nn.AvgPool2d(5, padding=(2), count_include_pad=True)
```

Comment:
I think this should be false.

Reply:
Re-writing this via 3 separate test cases for clarity - 1)avg_pool2d_with_count_include_pad_set_true; 2)avg_pool2d_with_count_include_pad_set_false; 3)avg_pool2d